### PR TITLE
Add bundled tests for "onInitialized" and "onDisposing" events

### PR DESCRIPTION
**Pull request by DokaRus**
https://trello.com/c/UMC7y9Us

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import GoogleStaticProvider from 'ui/map/provider.google_static';
+import fx from 'animation/fx';
+import { each } from 'core/utils/iterator';
+import 'bundles/modules/parts/widgets-all';
+
+const { module: testModule, test } = QUnit;
+
+testModule('Init and dispose', {
+    before: function() {
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js:9]: beforeEach?

File: testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    after: function() {
+        this.element.remove();
+    }
+}, function() {
+    fx.off = true;
+    GoogleStaticProvider.remapConstant('/mapURL?');
+
+    const excludedComponents = [
+        'dxLayoutManager'
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js]: Why do we need to exclude it?

File: testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js (hunk 1 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js:9]: Each test creates a widget and then destroys it. If the container cleaning is broken somewhere, which leads to an error, these tests will indicate that

File: testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js (hunk 2 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets/init_dispose_widget_bundled_tests.js]: There is no reason for now. Fixed it, thanks

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**